target_actionkit/sinks.py

```diff
@@ -49,6 +49,9 @@ def upsert_record(self, record: dict, context: dict):
             
             if existing_users:
                 user_id = existing_users[0].get("id")
+                # TODO:I got {"errors": {"zip": ["Data too long! Please use a smaller value, see the schema for the maximum allowed input"]}}}
+                # I'm not sure if this is the correct way to handle this, but it's a temporary fix
+                record.pop("zip", None)
```

Comment:
Doesn't this just mean you were providing an invalid `zip`?

Reply:
It was a valid one and in the schema there wasn't any valuable information on how to handle it... If I let it updating the zip it would break things.